.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: 🐛 Report a bug
+about: Create a bug report to help improve pynab
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Version of pynab:** v0.xx.xx
+
```

Comment:
It would be useful to also add the Raspberry Pi OS/kernel version (**Version** info from the **System Information** web page) to this template.